src/torchmetrics/image/ssim.py

```diff
@@ -55,9 +55,9 @@ class StructuralSimilarityIndexMeasure(Metric):
     Example:
         >>> from torchmetrics import StructuralSimilarityIndexMeasure
         >>> import torch
-        >>> preds = torch.rand([16, 1, 16, 16])
+        >>> preds = torch.rand([3, 3, 256, 256])
         >>> target = preds * 0.75
-        >>> ssim = StructuralSimilarityIndexMeasure()
+        >>> ssim = StructuralSimilarityIndexMeasure(data_range=1.0)
```

Comment:
why can't we stick to `None` here?

Reply:
I guess there's no reason why we couldn't since this is just a test case that we want to come out the same every time. Technically the range for `rand` is [0, 1) though